src/components/ec-badge/ec-badge.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <span
+    class="ec-badge"
+    data-test="ec-badge"
+    :class="{
+      [`ec-badge--${type}`]: type
+    }"
+  >
+    <slot v-bind="{ value }">{{ value }}</slot>
+  </span>
+</template>
+
+<script setup>
+
+defineProps({
+  type: {
+    type: String,
+    validator(value) {
+      return ['error', 'info', 'success', 'warning'].includes(value);
+    },
+    default: 'info',
+  },
+  value: {
+    type: [String, Number],
+    required: true,
+  },
+});
+</script>
+
+<script>
+export default {
+  name: 'EcBadge',
+  compatConfig: {
+    MODE: 3,
+  },
+};
+</script>
+
+<style>
+:root {
+  --ec-filter-badge-width: 16px;
+  --ec-filter-badge-height: 16px;
+}
+
+.ec-badge {
+  @apply tw-text-gray-8;
+  @apply tw-text-center tw-help-text;
+  @apply tw-py-0 tw-px-4;
+
+  min-width: var(--ec-filter-badge-width);
+  height: var(--ec-filter-badge-height);
+  border-radius: calc(var(--ec-filter-badge-width) / 2);
+
+  &--info {
+    @apply tw-bg-info;
```

Comment:
This one has to follow the current design, which uses `tw-bg-key-5` color.

This is also the root cause that the visual regression snapshots for date-range filter are different and needed an update. It's a different background color.

Reply:
ok, corrected it